css/main_2017.scss

```diff
@@ -0,0 +1,567 @@
+$light-fg: #444;
```

Comment:
There's no need to backup the old stuff. If you want you can make a branch, but for the most part it's really not needed since it's all in git.

There's also https://github.com/nwhacks/nwhacks2017_static which is hosting https://2017.nwhacks.io